File: phonetic.rb
#!/usr/bin/env ruby
require 'yaml'
require 'pathname'

hash = File.open(Pathname(__dir__)+'phonetic.yaml') {|yf| YAML::load(yf)}

string = ARGV[0].upcase.gsub(/[ \.]/,"").strip
output = Array.new()
string.each_char{|x|
  if hash.key?(x)
    output << hash[x]
  else
    output << x
  end
}
print output.join("\n")

File: phonetic.py
#!/usr/bin/env python3
import os
import sys
import yaml
from yaml.loader import SafeLoader

with open(os.path.join(sys.path[0],'phonetic.yaml')) as f:
    data = yaml.load(f, Loader=SafeLoader)
output = []
chars = list(sys.argv[1].strip().upper())
for x in chars:
    if x in data.keys():
        output.append(data[x])
    else:
        output.append(x)

print(" ".join(output))